src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,70 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { FC } from 'react';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage?: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: FC<Props> = ({
+  total, perPage, currentPage = 1, onPageChange,
+}) => {
+  const numberOfPages = Math.ceil(total / perPage);
+  const pages = [];
+
+  for (let i = 1; i <= numberOfPages; i += 1) {
+    pages.push(String(i));
+  }
+
+  const onFirstPage = currentPage === 1;
+  const onLastPage = currentPage === Number(pages.at(-1));
```

Comment:
Usually, we name our booleans with `is` prefix, `on` is reserved for handlers
[read more](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const isFirstPage = currentPage === 1;
  const isLastPage = currentPage === Number(pages.at(-1));
```